### PR TITLE
:sparkles: Nick command

### PR DESCRIPTION
**Pull request by Gilbert142**
**Please describe the changes this PR makes and why it should be merged:**

**What does this PR do:**
- [ ] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [x] This PR modifies commands
  - [ ] This PR changes command functions
  - [x] This PR changes metadata for the command, updated in `commands.js`
  - [x] This PR removes and/or adds commands
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (endpoint functions)
  - [ ] This PR modifies paths to existing resources
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: Commands/Public/nick.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+				return msg.send({
+					embed: {
+						color: Colors.SOFT_ERR,
+						title: `I'm sorry, but you can't do that... 😔`,
+						description: "Your nickname must not contain more than 32 characters!",
+					},
+				});
+			}
+
+			if (nickname.trim() === ".") nickname = "";
```
Comment (vladfrangu) [inline Commands/Public/nick.js]: nickname = null

File: Commands/Public/nick.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+			msg.send({
+				color: Colors.SUCCESS,
+				description: `You now have the nickname \`${msg.member.nickname}\` on this server 💥`,
+			});
+		}
+	} else if (msg.member.nickname) {
+		msg.send({
+			embed: {
+				color: Colors.RESPONSE,
+				description: `🏷 Your nick on this guild is \`${msg.member.nickname}\``,
```
Comment (vladfrangu) [inline Commands/Public/nick.js]: Emoji at the end

File: Commands/Public/nick.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
+const ArgParser = require("../../Modules/MessageUtils/Parser");
+
+module.exports = async ({ Constants: { Colors, Text }, client }, { serverDocument }, msg, commandData) => {
+	if (msg.suffix) {
+		let [target, nickname] = ArgParser.parseQuoteArgs(msg.suffix, "|");
+		if (target && nickname) {
+			if (!msg.member.hasPermission("MANAGE_NICKNAMES")) {
```
Comment (vladfrangu) [inline Commands/Public/nick.js]: permissions.has

File: Commands/Public/nick.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+			await member.edit({ nick: nickname }, `Command issued by ${msg.author.tag}`);
+			msg.send({
+				embed: {
+					color: Colors.SUCCESS,
+					description: `${client.getName(serverDocument, member, true)} now has ${nickname === "" ? "no nickname" : `the nickname \`${member.nickname}\``} on this guild 💥`,
+				},
+			});
+		} else if (target) {
+			nickname = target;
+			if (nickname === ".") nickname = "";
```
Comment (vladfrangu) [inline Commands/Public/nick.js]: Ditto, null

File: Commands/Public/nick.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+			if (!msg.member.manageable) {
+				return msg.send({
+					embed: {
+						color: Colors.SOFT_ERR,
+						title: `I'm sorry, but I can't do that... 😔`,
+						description: `I'm missing permissions to manage you!\nEither you are above me or I don't have the **Manage Nicknames** permission.`,
+					},
+				});
+			}
+			if (nickname.length > 32) {
```
Comment (vladfrangu) [inline Commands/Public/nick.js:92]: Why dont we truncate it manually to 32 chars?

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: Commands/Public/nick.js (hunk 5 above)
Comment (Gilbert142) [reply Commands/Public/nick.js:92]: Could lead to unexpected behavior for user. Better to throw error and allow user to adjust.

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**